core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ReadOnlyLDAPUserStoreManager.java

```diff
@@ -1914,7 +1914,6 @@ protected String getEffectiveSearchBase(boolean shared) {
                             membershipValue = escapeLdapNameForFilter(ldn);
                         }
                     } catch (InvalidNameException e) {
-                        log.error("Error while creating LDAP name from: " + nameInSpace);
                         throw new UserStoreException("Invalid naming exception for : " + nameInSpace, e);
```

Comment:
Removed unnecessary space in throw new UserStoreException("Invalid naming exception for : " + nameInSpace, e);